compiler/testData/codegen/bytecodeListing/fileClassWithPrivateDeclarationsOnly_before.kt

```diff
@@ -0,0 +1,8 @@
+// WITH_RUNTIME
+// IGNORE_BACKEND: JVM
```

Comment:
Please add explicit `// !LANGUAGE: -PackagePrivateFileClassesWithAllPrivateMembers` here for clarity. :)